inst/templates/ad_adae.R

```diff
@@ -40,7 +40,7 @@ adae <- ae %>%
   derive_vars_merged(
     dataset_add = adsl,
     new_vars = adsl_vars,
-    by = exprs(STUDYID, USUBJID)
+    by = exprs(!!!get_admiral_option("subject_keys"))
```

Comment:
> Also, I believe in this case you could do `by_vars = get_admiral_option("subject_keys")` directly rather than quoting and unquoting `by_vars = exprs(!!!get_admiral_option("subject_keys"))` (this is applicable in lots of other places as well.) You'd only need to use `exprs` and unquoting when you want to use other by vars as well, for instance `by_vars = exprs(!!!get_admiral_option("subject_keys"), AEDECOD)` - @bundfussr do you agree?

I agree.